<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProductoItem</class>
 <widget class="QWidget" name="ProductoItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>248</width>
    <height>138</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="pushButton_ing_tipo">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>INGRESAR TIPO</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_ing_marca">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>INGRESAR MARCA</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_ing_unidad">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>INGRESAR UNIDAD</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QPushButton" name="pushButton_bus_tipo">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>BUSCAR TIPO</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_bus_marca">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>BUSCAR MARCA</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_bus_unidad">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>BUSCAR UNIDAD</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>98</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_salir">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Salir</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>pushButton_ing_tipo</tabstop>
  <tabstop>pushButton_bus_tipo</tabstop>
  <tabstop>pushButton_ing_marca</tabstop>
  <tabstop>pushButton_bus_marca</tabstop>
  <tabstop>pushButton_ing_unidad</tabstop>
  <tabstop>pushButton_bus_unidad</tabstop>
  <tabstop>pushButton_salir</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
